openapi/p4pa-debt-position.openapi.yaml

```
@@ -130,11 +130,14 @@ paths:
           description: Invalid request
         "500":
           description: Internal server error
-  /installments/{organizationId}/export:
+  /export/organization/{organizationId}/installments/paid:
     get:
       tags:
         - installment
```

if there are multiple export API, which would share the same validation (eg: time interval), probably it could be useful to use the same tag
```suggestion
        - data-exports
```